ethcore/res/ethereum/constantinople_test.json

```diff
@@ -1,16 +1,16 @@
 {
-	"name": "Byzantium (Test)",
+	"name": "Constantinople (test)",
 	"engine": {
 		"Ethash": {
 			"params": {
 				"minimumDifficulty": "0x020000",
 				"difficultyBoundDivisor": "0x0800",
 				"durationLimit": "0x0d",
-				"blockReward": "0x29A2241AF62C0000",
+				"blockReward": "0x1BC16D674EC80000",
 				"homesteadTransition": "0x0",
 				"eip100bTransition": "0x0",
 				"difficultyBombDelays": {
-					"0": 3000000
+					"0": 8000000
```

Comment:
Why 8_000_000? What about

```
				"difficultyBombDelays": {
					"0": 3000000
					"5": 2000000
				}
```

to match eip 649, 1234?

Reply:
I think those json test are used in a all or nothing basis (eg from ethereum test), so putting the sum of eip 649 and 1234 together at 0 makes sense (on two line it would be better but I think json dont allow to identical keys). I got a doubt 649 was a 3 million delay and 1234 another additional 5 million ?

Reply:
fine by me. 649 is 3 million, 1234 is 2 million. sum is 5 million.

Reply:
My mistake, when reading 1234, " 5 million blocks later than previously specified with the Homestead fork" gave me the wrong impression of additional 5M but it would have been only true if refering to byzantium fork. 
So yes we need to use 5M instead of 8M for this line.